src/app/partials/nav/header-static/header-static.component.scss

```diff
@@ -3,19 +3,299 @@
+	box-shadow: 0px 15px 20px $overlay-light;
+}
+@mixin shadow-header-large {
+	box-shadow: 0px 0px 5px $overlay-light;	
+}
+
+.dark-overlay{
+	width:100%;
+	height:100%;
+	position:fixed;
+	z-index:100;
+	background:$overlay-light;
+	overflow:hidden;
+}
+
+.menu-expander{
+	display:none;
+	color: black;
+	cursor:pointer;
+	position:absolute;
+    top:25px;
+    right:10px;
+    z-index:100;
+
+}
+
+.menu-bar1, .menu-bar2, .menu-bar3 {
+    width: 30px;
+    height: 5px;
+    
+    margin: 6px 0;
+    transition: 0.4s;
+}
+
+.dark-bg {
+	background: black;
+}
+
+.white-bg {
+	background: white;
+}
+
+.menu-change {
+	.menu-bar1 {
+	   -webkit-transform: rotate(-45deg) translate(-9px, 6px);
+    	transform: rotate(-45deg) translate(-9px, 6px);
+    }
+}
+
+.menu-change {
+	.menu-bar2 {
+		opacity: 0;
+	}
+}
+
+.menu-change {
+	.menu-bar3 {
+	    -webkit-transform: rotate(45deg) translate(-8px, -8px);
+    	transform: rotate(45deg) translate(-8px, -8px);
+    }
+}
+
+.header-shadow{
+	@include shadow-btn;
+}
+
+.header-shadow-large{
+	@include shadow-header-large;
+}
+
+.dark-text{
+	color:$gray-darker !important;
+}
+
+.invert-color{
+	filter: invert(100%);
+}
+
+.logo-container {
+	text-align: right;
+	display:inline-block;
+	float: left;
+	width:10%;
+	margin-top:30px;
+	margin-left:30px;
+	img:hover {
+		cursor:pointer;
+	}
+	img {
+		height:40px;
+	}
+}
+
+.navlinks-container {
+	display:inline-block;
+	float: right;
+	width:70%;
+	position:relative;
+	text-align:right;
+	transition:height 0.5s ease;
+	padding-bottom:10px;
+	padding-right:10px;
+	div{
+		display:block;
+		position:relative;
+		margin-top:20px;
+		padding-top:10px;
+		height:100%;
+		margin-right:20px;
+	}
+	ul {
+		list-style: none;
+		margin-top:30px;
+		li {
+			display: inline;
+		}
+	}
+}
+
+.btn-nofill{
+    padding:10px 30px 10px 30px;
+    border:1.5px solid white;
+    border-radius:20px;
+    margin-left:5px;
+    margin-bottom:5px;
+    color:white;
+    text-align: center;
+    cursor:pointer;
+    @include shadow-btn;
+    &:hover {
+	  background: $gray-light;
+	  color: black;
+	  transition:all 0.5s ease;
+	  @include shadow-btn-large;
+	}
+}
+
+.btn-fire {
+	display:inline-block;
+	@include fire(false);
+    border: none;
+    color: white;
+    padding:13px 33px 13px 33px;
+ 	border-radius:20px;
+    margin-left:5px;
+    margin-bottom:5px;
+    margin-top:0 !important;
+    margin-bottom: 0 !important;
+    text-align: center;
+    cursor:pointer;
+    @include shadow-btn;
+    &:hover {
+	@include fire(true);
+		@include shadow-btn-large;
+	 	transition:all 0.5s ease;
+	}
+}
+
+.btn-fill {
+    padding:13px 33px 13px 33px;
+    border-radius:20px;
+    margin-left:5px;
+    margin-bottom:5px;
+    background:white;
+    color:$blue-dark;
+    text-align: center;
+    cursor:pointer;
+    @include shadow-btn;
+    &:hover {
+    	@include shadow-btn-large;
+		background: $gray-light;
+		color: black;
+		transition:all 0.5s ease;
+	}
+}
+
```

Comment:
From all button classes  `btn-*` move common css to common class and named as `btn`.
Like ```
.btn{
  border-radius: 40px;
}```
and add class like... `btn btn-fill` @live-wire

Reply:
👍 
Moved the button css to `base.scss`